src/DomDocuments/BaseDocument.php

```diff
@@ -156,13 +156,24 @@ protected function appendPerformanceTypeFields(\DOMElement $element, $object): v
      * Use this instead of createElement().
      *
      * @param string $tag
-     * @param string $textContent
+     * @param string|null $textContent
+     * @param $object
+     * @param array|null $attributes
```

Comment:
Can you add how this argument would be used? Maybe rename to `$methodToAttributeMap`?

Reply:
Argument is used as follows:

`$this->createNodeWithTextContent('billable', $quantity->getBillableToString(), $quantity, array('locked' => 'getBillableLockedToString'))`

Method renamed